wkcuber/wkcuber/convert_image_stack_to_wkw.py

```diff
@@ -68,32 +66,20 @@ def main(args: Namespace) -> None:
         args.start_z,
         args.pad,
         args,
-    )
-
-    write_webknossos_metadata(
-        args.target_path,
-        args.name,
         args.scale,
-        compute_max_id=False,
-        exact_bounding_box=bounding_box,
     )
 
     if not args.no_compress:
         compress_mag_inplace(args.target_path, args.layer_name, Mag(1), args)
```

Comment:
Shouldn't this be `args.target_mag` instead of `Mag(1)`? (Same few lines below)

Reply:
@youri-k can you comment on this?